taskvine/test/TR_vine_python_unlink_when_done.sh

```diff
@@ -15,7 +15,7 @@ PORT_FILE=vine.port
 check_needed()
 {
 	[ -n "${CCTOOLS_PYTHON_TEST_EXEC}" ] || return 1
-	"${CCTOOLS_PYTHON_TEST_EXEC}" -c "import cloudpickle"  || return 1
+	"${CCTOOLS_PYTHON_TEST_EXEC}" -c "import cloudpickle; import dask"  || return 1
```

Comment:
Do we need to get these deps into our CI environment?

Reply:
Probably in the cctools-package-test?